docs/reference/commandline/run.md

```diff
@@ -58,8 +58,8 @@ parent = "smn_cli"
                                     'bridge': creates a new network stack for the container on the docker bridge
                                     'none': no networking for this container
                                     'container:<name|id>': reuses another container network stack
-                                    'host': use the host network stack inside the container
-                                    'NETWORK': connects the container to user-created network using `docker network create` command
+                                    'host': uses the host network stack inside the container
+                                    'NETWORK_NAME_or_ID': connects the container to user-defined network using `docker network create` command
```

Comment:
for reference, the `docker network rm` command currently also uses `NETWORK` but mentions that both "name" and "id" can be used; https://github.com/docker/docker/blob/master/docs/reference/commandline/network_rm.md